examples/scripts/DFN_high_Crate.py

```diff
@@ -0,0 +1,48 @@
+#
+# Example showing how to load and solve the DFN
+#
+
+import pybamm
+import numpy as np
+
+pybamm.set_logging_level("INFO")
+
+# load model
+model = pybamm.lithium_ion.DFN()
+
+# create geometry
+geometry = model.default_geometry
+
+# load parameter values and process model and geometry
+param = model.default_parameter_values
+param.update(
+    {
+        "C-rate": 50,
+        "Current function": "[constant]",
+    }
+)
+param.process_model(model)
+param.process_geometry(geometry)
+
+model.events.pop("Zero electrolyte concentration cut-off")
+
+# set mesh
+var = pybamm.standard_spatial_vars
+var_pts = {var.x_n: 30, var.x_s: 30, var.x_p: 30, var.r_n: 10, var.r_p: 10}
+mesh = pybamm.Mesh(geometry, model.default_submesh_types, var_pts)
+
+# discretise model
+disc = pybamm.Discretisation(mesh, model.default_spatial_methods)
+disc.process_model(model)
+
+# solve model
+t_eval = np.linspace(0, 0.2, 1E4)
```

Comment:
needs to be updated with t_eval in seconds

Reply:
I did this such a long time ago that I thought someone else had done it haha. I have removed it now.